external-crates/move/crates/move-compiler/src/shared/mod.rs

```diff
@@ -866,3 +868,124 @@ macro_rules! process_binops {
 }
 
 pub(crate) use process_binops;
+
+//**************************************************************************************************
+// Virtual file system support
+//**************************************************************************************************
```

Comment:
minor nit, move this near the IndexedPackagePath

Reply:
I think it would a bit weird to have this right at the top of the file so I moved `IndexedPackagePath` down instead - happy to reorganize differently if need be, though!

Reply:
Works for me!